### PR TITLE
feat: add bmc url

### PR DESCRIPTION
**Pull request by HappyFX**
@SuperQ Hi!
Please review the PR. I need to add URL info from BMC.

Actual output from console

```bash
# bmc-info | egrep "^BMC URL\s*:\s*(.*)"
BMC URL                       : https://123.234.21.231:443
```

### CODE REVIEW

**Comment by HappyFX** [comment]
@SuperQ Hi!
Please review the PR. I need to add URL info from BMC.

Actual output from console

```bash
# bmc-info | egrep "^BMC URL\s*:\s*(.*)"
BMC URL                       : https://123.234.21.231:443
```

**Comment by HappyFX** [comment]
Hello @bitfehler @SuperQ, can you take a look on my PR?

File: collector_bmc.go [hunk 1]
```diff
@@ -63,11 +63,17 @@ func (c BMCCollector) Collect(result freeipmi.Result, ch chan<- prometheus.Metri
 		logger.Debug("Failed to parse bmc-info data", "target", targetName(target.host), "error", err)
 		systemFirmwareVersion = "N/A"
 	}
+	bmcUrl, err := freeipmi.GetBMCInfoBmcUrl(result)
+	if err != nil {
+		// This one is not always available.
+		logger.Debug("Failed to parse bmc-info data", "target", targetName(target.host), "error", err)
+		systemFirmwareVersion = "N/A"
```
Comment (bitfehler) [inline collector_bmc.go:70]: This should set `bmcUrl`, not `systemFirmwareVersion`

**Review by bitfehler — CHANGES_REQUESTED** [review]
Thanks. Looks good, except for the one issue

File: collector_bmc.go (hunk 1 above)
Comment (HappyFX) [reply collector_bmc.go]: Thank you! Fixed

**Pull request merged**